dashboard/config/scripts_json/coursec-2021.script_json

```diff
@@ -1914,7 +1914,7 @@
+        "description": "This short activity will help students relate the ideas of persistence and debugging to the puzzles that they are about to complete online.\n\n**Display:** Project a copy of the **Course C, Lesson 2 Maze Bridging Page - Puzzle Manipulative (PDF)** for the class to see.  Make sure that you have pre-placed the movement blocks in the workspace using **Unplugged Maze Blocks - Manipulatives** in a configuration like the one below:\n\n![](https://images.code.org/ac052e9d25e77dd800fbf18871bc5fcf-image-1611178828427.47.39.png)\n\n**Model:**  Tell students that you have this workspace on display that looks just like the area that they will see when they start to do the Code.org puzzles online.  As the teacher, let them know that you are SO SMART that you already put all of the code in that you are going to need to solve this puzzle, then ask them to watch you \"Run\" it by moving your finger (or a penny, or some other indicator) along the path.\n\nIt won't be long before you run into a block of TNT.  Feign frustration.\n\n**Discuss:**\n\n- What am I feeling right now, do you think?\n- Should I quit?\n- Should I throw all of the code away and start over?\n\n**Think:** How can I fix this program so that I don't run into the TNT?\n\n**Pair:** Have students work on solutions to get the bird around the TNT. Depending on your classroom, you might want to either have them fix each mistake one at a time (with demos in between) or students might feel comfortable working together to fix the entire program.\n\n**Share:** Have volunteers come up to help move the blocks into the right location.  \"Run\" the program over and over as a class, fixing bugs, until the bird does what it is supposed to.  Continue to point out experiences that relate to persistence, frustration, and debugging.\n\nWhen your class reaches the pig, celebrate not only their achievements, but their persistence!",
```

Comment:
I had to use a different tool to see what changed on this line:

![Screen Shot 2021-01-21 at 4 20 08 PM](https://user-images.githubusercontent.com/8001765/105428637-9dae4300-5c04-11eb-9434-eb89d5ac8a39.png)

are there next steps that need to be tracked for this image, whose width is being removed?

Reply:
nope; in this case, the image being used never needed to be a size other than its display size, so I just resized the actual image. See the commit description https://github.com/code-dot-org/code-dot-org/pull/38669/commits/79103952dc26fc2e6bd702418a1e0aae8c518a1e